### PR TITLE
[Doc] Add coding style guide and how to contribute guide

### PR DESCRIPTION
**Pull request by acezen**
## Related issue number

Fixes

### CODE REVIEW

**Comment by sighingnow** [comment]
Can you given some instructions to tell how to format corresponding code to those styles, e.g., `make gsa_clformat`, respectively.

File: docs/development/code_style_guide.md [hunk 1]
```diff
@@ -1,17 +1,90 @@
...
+    $ files_to_format=$(git ls-files *.java)
+    $ java -jar google-java-format-1.13.0-all-deps.jar --aosp --skip-javadoc-formatting -i $files_to_format
+```
+
+## Conclusion
+
+In conclusion, following a consistent code style guide is crucial for maintaining code quality and readability. By adhering to these guidelines, we can ensure that code in GraphScope follows best practices and is easy to read, understand, and maintain. It is important to note that these guidelines are not set in stone and may evolve over time as the project grows and changes. Therefore, we encourage all contributors to review and suggest improvements to this document as needed.
+
+
+[1]: https://google.github.io/styleguide/cppguide.html
```
Comment (yecol) [inline docs/development/code_style_guide.md:80]: make all the links inline and clickable.

File: docs/development/code_style_guide.md [hunk 2]
```diff
@@ -1,17 +1,90 @@
...
+
+For Java:
+
+```bash
+    $ wget https://github.com/google/google-java-format/releases/download/v1.13.0/google-java-format-1.13.0-all-deps.jar
+    $ files_to_format=$(git ls-files *.java)
+    $ java -jar google-java-format-1.13.0-all-deps.jar --aosp --skip-javadoc-formatting -i $files_to_format
+```
+
+## Conclusion
```
Comment (yecol) [inline docs/development/code_style_guide.md:75]: No need for conclusion.

File: docs/development/code_style_guide.md [hunk 3]
```diff
@@ -1,17 +1,90 @@
...
 
+We follow the [Google C++ Style Guide][1] for coding standards in C++.
+
 ## Python Style
 
+We follow the [black][2] code style for Python coding standards.
+
 ## Rust Style
 
+We follow a [custom style][3] for coding standards in Rust.
```
Comment (yecol) [inline docs/development/code_style_guide.md:17]: what?

File: docs/development/how_to_contribute.md [hunk 4]
```diff
@@ -1,4 +1,184 @@
...
-(This document is intended to help you get started contributing to GraphScope project.)
\ No newline at end of file
+
+### DingTalk
+
+TBF
+
+## Bug Report
+
+If you find a bug in GraphAr, first make sure that you are testing against the [latest version of GraphScope](https://github.com/alibaba/GraphScope/tree/main),
```
Comment (yecol) [inline docs/development/how_to_contribute.md:32]: GraphAr -> GraphScope.

File: docs/development/how_to_contribute.md [hunk 5]
```diff
@@ -1,4 +1,184 @@
...
+have been added because our users saw the need.
+
+[Open a feature request issue](https://github.com/alibaba/GraphScope/issues/new?assignees=&labels=&template=feature_request.md&title=) on GitHub which describes the feature you would
+like to see, why you need it, and how it should work.
+
+## Improve Documentation
+
+A great way to contribute to the project is to improve documentation. If you found some docs to be incomplete or inaccurate, share your hard-earned knowledge with the rest of the community.
+
+Documentation improvements are also a great way to gain some experience with our submission and review process, discussed below, without requiring a lot of local development environment setup. In fact, many documentation-only changes can be made directly in the GitHub web interface by clicking the “edit” button. This will handle making a fork and a pull request for you.
```
Comment (yecol) [inline docs/development/how_to_contribute.md:51]: in the Github web? no, we have a more straight-forward way.  we have the edit button on the docs page.

File: docs/development/how_to_contribute.md [hunk 6]
```diff
@@ -1,4 +1,184 @@
...
+
+If you find a bug in GraphAr, first make sure that you are testing against the [latest version of GraphScope](https://github.com/alibaba/GraphScope/tree/main), 
+your issue may already have been fixed. If not, search our [issues list](https://github.com/alibaba/GraphScope/issues)
+on GitHub in case a similar issue has already been opened.
+
+If you get confirmation of your bug, [file a bug issue](https://github.com/alibaba/GraphScope/issues/new?assignees=&labels=&template=bug_report.md&title=%5BBUG%5D+) before starting to code. When submitting an issue, please include a clear and concise description of the problem, any relevant code or error messages, and steps to reproduce the issue.
+
+## Request Feature
+
+If you find yourself wishing for a feature that doesn't exist in GraphScope, you are probably not alone.
```
Comment (yecol) [inline docs/development/how_to_contribute.md:40]: ... Maybe proofread in chatgpt. or Cursor.

**Review by yecol — CHANGES_REQUESTED** [review]

**Review by yecol — APPROVED** [review]

**Pull request merged**